integrationTests/src/test/resources/AndroidTestingBlueprint/build.gradle

```diff
@@ -27,11 +26,10 @@ apply plugin: "com.dorongold.task-tree"
 
 allprojects {
     repositories {
-        google()
         jcenter()
-		    maven {
-		        url "https://maven.google.com"
-		    }
+        maven {
+		    url "https://maven.google.com"
+		}
```

Comment:
The lines have mixed tabs and spaces. Would you please mind to normalize them? (e.g. use spaces as in other lines)